<launch>
    <arg name="device_index" />
    <arg name="device_name" />
    <arg name="sample_rate" default="16000"/>
    <arg name="num_channels" default="1"/>
    <arg name="sample_width" default="2"/>
    <arg name="frames_per_buffer" default="1024"/>
    <node
      pkg="audio"
      name="$(anon capture)"
      type="capture.py"
      output="screen"
      >
        <param    name ="device_index" value="$(arg device_index)" />
        <param    name="device_name" value="$(arg device_name)" />
        <param    name="sample_rate" value="$(arg sample_rate)" />
        <param    name="num_channels" value="$(arg num_channels)" />
        <param    name="sample_width" value="$(arg sample_width)" />
        <param    name="frames_per_buffer" value="$(arg frames_per_buffer)" />

    </node>
</launch>
